papaparse.js

```diff
@@ -554,6 +560,11 @@ License: MIT
 			return results;
 		};
 
+		this.halted = function()
```

Comment:
Why creating a function? Why not accessing the attribute directly?

Reply:
It feels peculiar to access data members of a different class directly. I generally work with TS or ES6. I can change that if you wish.